CMakeLists.txt

```diff
@@ -104,7 +104,7 @@ option(Kokkos_ENABLE_DEPRECATION_WARNINGS "Enable Kokkos deprecation warnings" O
 include(FetchContent)
 FetchContent_Declare(kokkos
   GIT_REPOSITORY https://github.com/kokkos/kokkos.git
-  GIT_TAG        3.6.00
+  GIT_TAG        3.7.00
```

Comment:
Is there a specific reason for `3.7.00` instead of the latest one `3.7.01`?

Reply:
Good spot, Chae-Yeun! This was updated a while ago and let me update it.